src/libraries/System.Private.CoreLib/src/System/Number.NumberToFloatingPointBits.cs

```diff
@@ -68,11 +96,19 @@ public FloatingPointInfo(ushort denormalMantissaBits, ushort exponentBits, int m
 
                 InfinityBits = infinityBits;
                 ZeroBits = 0;
+
+                LargerPowerOfTen = largerPowerOfTen;
+                SmallerPowerOfTen = smallerPowerOfTen;
+
+                InfinitePower = infinitePower;
+
+                MinExponentRoundToEven = minExponentRoundToEven;
+                MaxExponentRoundToEven = maxExponentRoundToEven;
+
+                MaxExponentFastPath = maxExponentFastPath;
             }
         }
-
-        private static readonly float[] s_Pow10SingleTable = new float[]
-        {
+        private static ReadOnlySpan<float> s_Pow10SingleTable => new float[]{
```

Comment:
In the original, this was
```

    internal readonly static double[] powers_of_ten_double = { 1e0,  1e1,  1e2,  1e3,  1e4,  1e5,  1e6,  1e7,  1e8,  1e9,  1e10, 1e11,
                                              1e12, 1e13, 1e14, 1e15, 1e16, 1e17, 1e18, 1e19, 1e20, 1e21, 1e22};

    internal readonly static float[] powers_of_ten_float = { 1e0F, 1e1F, 1e2F, 1e3F, 1e4F, 1e5F,
                                            1e6F, 1e7F, 1e8F, 1e9F, 1e10F};
```

https://github.com/CarlVerret/csFastFloat/blob/master/csFastFloat/Constants/Constants.cs

At a glance, this looks here like heap allocation.